.drone.yml

```diff
@@ -17,7 +17,14 @@ steps:
       event: push
     commands:
       - yarn lint
-      -
+
+  - name: Type check
```

Comment:
btw, I hate that you capitalized the first letter here, we sort of have a schema going on y'know 😣